contributors/DEVELOPER-GUIDE.md

```diff
@@ -55,19 +55,27 @@ Once you're happy with the file contents, you'd need to reverse-engineer the cha
 
 For example, `generated.txt` would have a sibling `generated.txt.dev` file with information about the "template" and "args" files that contributed to the current result.
 
-## Back-merging main branch / Publishing to NPM (TODO: Update to main branch workflow)
+## Back-merging main branch / Publishing to NPM
 
-1. Make sure you have the latest changes from `main` branch
-2. Checkout to `cli` branch and create a new branch from it eg: `cli-backmerge`
-3. Checkout to `cli-backmerge` branch and `git merge main`
-4. If there are any conflicts, resolve them and commit the changes
-5. Add changeset by doing `yarn changeset add` follow prompt and commit changes, learn more about changeset [here](https://github.com/scaffold-eth/scaffold-eth-2/blob/cli/CONTRIBUTING.md#changeset)
-6. Push the branch and create a PR against `cli` branch
+1. Sync `upstream-main` branch with SE-2 in the GitHub UI, using "Sync fork" (you can also add the remote branch locally and do it locally)
```

Comment:
Maybe start with: Switch to `upstream-main` branch and sync it with SE-2 repo from GitHub UI, ...

Reply:
Updated!